File: ios/JJSBlurView.m
//
//  BlurView.m
//  react-native-blurview
//
//  Created by Juan J LF on 9/26/21.
//


#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"
#import <React/RCTBridgeModule.h>
#import <UIKit/UIKit.h>


@interface
RCT_EXTERN_MODULE(JJSBlurView,RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(blurNode, NSString)


@end

File: ios/JJSBlurView.swift
//
//  BlurView.swift
//  react-native-blurview
//
//  Created by Juan J LF on 9/26/21.
//

import Foundation
import UIKit

@objc(JJSBlurView)
class JJSBlurView: RCTViewManager {
    
    static var MaskViews = [Int]()

    override func view() -> UIView! {
       return BlurView()
     }
   override class func requiresMainQueueSetup() -> Bool {
        return true
    }
}
